third_party/googletest/src/googletest/src/gtest.cc

```diff
@@ -1152,7 +1152,9 @@ class Timer {
 // Timer instead.
 TimeInMillis GetTimeInMillis() {
 #if GTEST_OS_STARBOARD
-  return SbTimeGetNow() / kSbTimeMillisecond;
+  EzTimeValue time_value;
```

Comment:
This perhaps deserves a comment as to whats going on in here - same one as in commit messages is fine

Reply:
Done